### ISSUE TITLE
[Package Issue]: Ghisler.TotalCommander

### ISSUE THREAD

**Issue by Scavy** [microsoft/winget-pkgs#115556]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installer hash mismatch.

### Brief description of your issue

The hash in the manifest is wrong for the x64 package.
https://github.com/microsoft/winget-pkgs/blob/master/manifests/g/Ghisler/TotalCommander/11.00/Ghisler.TotalCommander.installer.yaml

Current value in the manifest: 085D171D8A9AF4B56EBD90CE23E3C5C5B092470E338CC5333B7061B0F20828CD

The actual hash of the downloaded file: 9928f1fc20924d3ab3f9edb958b7205c1982077eeb5b1f10fa7072dad358a104

Is it supported to do a PR directly on this change myself?

### Steps to reproduce

Winget upgrade Ghistler.TotalCommander

### Actual behavior

(2/3) Found Total Commander [Ghisler.TotalCommander] Version 11.00
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://totalcommander.ch/win/tcmd1100x64.exe
  ██████████████████████████████  6.18 MB / 6.18 MB
Installer hash does not match.

### Expected behavior

(2/3) Found Total Commander [Ghisler.TotalCommander] Version 11.00
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://totalcommander.ch/win/tcmd1100x64.exe
  ██████████████████████████████  6.18 MB / 6.18 MB
Successfully verified installer hash
Starting package install...
Successfully installed

### Environment

```raw
winget --info
Windows Package Manager v1.5.1881
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.22621.1992
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.20.1881.0

Winget Directories
-----------------------------------------------------------------------------------------------------------------------
Logs                               %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\Diag…
User Settings                      %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\sett…
Portable Links Directory (User)    %LOCALAPPDATA%\Microsoft\WinGet\Links
Portable Links Directory (Machine) C:\Program Files\WinGet\Links
Portable Package Root (User)       %LOCALAPPDATA%\Microsoft\WinGet\Packages
Portable Package Root              C:\Program Files\WinGet\Packages
Portable Package Root (x86)        C:\Program Files (x86)\WinGet\Packages

Links
---------------------------------------------------------------------------
Privacy Statement   https://aka.ms/winget-privacy
License Agreement   https://aka.ms/winget-license
Third Party Notices https://aka.ms/winget-3rdPartyNotice
Homepage            https://aka.ms/winget
Windows Store Terms https://www.microsoft.com/en-us/storedocs/terms-of-sale

Admin Setting                             State
--------------------------------------------------
LocalManifestFiles                        Disabled
BypassCertificatePinningForMicrosoftStore Disabled
InstallerHashOverride                     Disabled
LocalArchiveMalwareScanOverride           Disabled
```

**Comment by mdanish-kh** [comment]
Close with reason: Resolved in #115597

**Comment by mdanish-kh** [comment]
Resolved in #115597
